meta-lmp-base/recipes-support/lmp-device-auto-register/lmp-device-auto-register/lmp-device-auto-register

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+
+TOKEN_FILE=${TOKEN_FILE-/etc/lmp-device-register-token}
+
+if [ ! -f $TOKEN_FILE ] ; then
+	echo "ERROR: Missing token file $TOKEN_FILE"
+	exit 1
+fi
+
+TOKEN=$(cat $TOKEN_FILE)
```

Comment:
Since you will use this value when calling lmp-device-register, probably better to make sure you are just getting the first line here, to avoid issues in case it ends up being a multi line file.